h5netcdf/legacyapi.py

```diff
@@ -1,8 +1,28 @@
+import sys
+
 import h5py
+import numpy as np
 
 from . import core
 
 
+def _check_return_dtype_endianess(endian="native"):
```

Comment:
@shoyer Do you have any suggestion, how to beautify this function. It looks a bit clunky to me.